openhtf/plugs/user_input.py

```diff
@@ -92,7 +93,10 @@ def run(self):
         if sys.stdin.isatty():
           termios.tcflush(sys.stdin, termios.TCIFLUSH)
 
-        line = ''
+        # Although this isn't threadsafe with do_setlocale=True, it doesn't work without it.
+        encoding = locale.getpreferredencoding(do_setlocale=True)
```

Comment:
Since this isn't threadsafe, and ConsolePrompt is a thread, maybe we can do it once a the module level during import and store the result in an `ENCODING` module global (locale is conceptually global state anyway)? That is probably safer than running it here in a thread.